draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2654,8 +2688,13 @@ for further discussion.
 Each encoding type ({{roadmap}}) accepts an arbitrary byte string and maps
 it to a point on the curve sampled from a distribution that depends on the
 encoding type.
-It is important to note that directly evaluating the mappings of {{mappings}}
-produces an output that is distinguishable from random.
+It is important to note that using a nonuniform encoding and/or directly
+evaluating one of the mappings of {{mappings}} produces an output that is
+easily distinguished from a random point.
+Protocols that use a nonuniform encoding SHOULD carefully analyze the security
+implications of using this encoding.
+When the required encoding is not clear, applications SHOULD use a random
```

Comment:
Unrelated to this change, but we should probably unify on terminology regarding the callers of hash-to-curve: we use applications and protocols in different places. I prefer applications, as it's more general and includes protocols.

Reply:
Good point. Opened #273.